IvanKorzh/HW5/2.py

```diff
@@ -0,0 +1,51 @@
+def long_fun():
```

Comment:
look at this implementation
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])